projects/material-addons/src/lib/stepper/step-header/step-header.component.spec.ts

```diff
@@ -0,0 +1,96 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { StepHeaderComponent } from './step-header.component';
+import { FocusMonitor, FocusOrigin } from '@angular/cdk/a11y';
+import { DebugElement } from '@angular/core';
+import { MatIconModule } from '@angular/material/icon';
+import { STEP_STATE } from '@angular/cdk/stepper';
+import { By } from '@angular/platform-browser';
+
+describe('StepHeaderComponent', () => {
```

Comment:
Why don't we use Cypress here?

Reply:
we are using Cypress in stepper.component.cy.ts. It contains tests for header